tests/run/utf16encoded.flags

```diff
@@ -0,0 +1 @@
+-encoding UTF16
```

Comment:
@Smarter That makes me realize this doesn't actually have to be a run test, as it won't compile anyway if it doesn't have the -encoding flag set to UTF-16. Should it be moved?